util/session/sessionmanager.go

```diff
@@ -32,7 +37,7 @@ func MakeSessionManager(secretKey []byte) SessionManager {
 }
 
 // Create creates a new token for a given subject (user) and returns it as a string.
-func (mgr SessionManager) Create(subject string) (string, error) {
+func (mgr SessionManager) create(subject string) (string, error) {
```

Comment:
`Create` method is used in argo-cd/server/session/session.go . Code does not compile because of this change